### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by JacoFourie** [arduino-org/arduino-firmware-wifilink#30]
Hi. 

I am sorry if this has been answered. Does this firmware support MQTT and the rest stuff like the Arduino version ? I have a lot of issues with the Arduino firmware and it seems there is not a lot of support for it. The board will work fine for a while and then it will start to drop its MQTT connection and connect again. This keeps on happening. It will also fall back to AP-STA mode even if I set it to STA mode. I have to reboot the wifi en set it back to STA mode. Then it will be stable for a while again. Does this firmware solve that issue ?

Regards.

**Comment by jandrassy** [comment]
Hi. Microntroler counterpart of this WiFi Link firmware is the Arduino WiFi Link library. WiFiLink.h is similar to Wifi and Ethernet Arduino libraries and it doesn't contain Ciao inspired 'connectors' like REST and MQTT.

Your board is Uno Wifi Dev. Ed? The original firmware of it is an older fork of esp-link firmware/library. Current version of the esp-link works on uno wifi dev. ed with exception of mcu ota.

**Comment by JacoFourie** [comment]
Yes I have the Uno Wifi Dev units. So if I understand you correctly this firmware does not have MQTT.
Do you know of firmware that has the MQTT interface but that is being supported better that the Arduino one as at this stage it is a real pain and if I can not find a fix for the firmware I will have to move to another board.

**Comment by jandrassy** [comment]
https://github.com/jeelabs/esp-link/releases

**Comment by JacoFourie** [comment]
Thank you very much !!
